programs/server/config.xml

```diff
@@ -1480,6 +1489,9 @@
         <column_family_options>
             <num_levels>2</num_levels>
         </column_family_options>
+        <block_based_table_options>
+            <block_size>1024</block_size>
```

Comment:
What will happen if user changes this setting for existing database? Will we get an error in `StorageEmbeddedRocksDB::initDB` or changes are supported by rocks db?

Reply:
No, they will be applied for new parts only, everything will work (and you can do manual compact to force update - #56225 )